tools/extractors/extractors.cpp

```diff
@@ -314,6 +314,27 @@ const std::vector<int> InitialGameStateExtractor::buildingFunctionDetectionWeigh
 	100, 100, 100, 100, 100, 100, 100, 100, 100
 };
 
+// Data is pulled from UFO2P, bld_cost_struc (was hard to determine common offset)
```

Comment:
for the other data tables we actually map in the .exe and read it directly in the extract step - can that be done here?

Reply:
I couldn't figure out exact structure and offset. We'll see if @Skin36 can find it, already asked for help :)